src/lib/initialValues.ts

```diff
@@ -43,28 +43,19 @@ export const employmentUserInputInitialValues: EmployedUserInput = {
   r039: '',
 }
 
-export const emptyChild = {
+export const makeEmptyChild = (): ChildInput => ({
+  id: Date.now(),
```

Comment:
Toto som este nevidel, je to good practice? :D

Reply:
Potrebujeme unikatne ID pre kazde dieta ale iba pre potreby renderovania. Tak preco nie, nevidim v tom nejaky problem 😃